assets/company/logo-fandom-powered-by-wikia.svg

```diff
@@ -0,0 +1,14 @@
+<svg width="254" height="26" viewBox="0 0 254 26" xmlns="http://www.w3.org/2000/svg">
+  <defs>
+    <linearGradient x1="0%" y1="100%" x2="63.848%" y2="32.54%" id="a">
```

Comment:
Let's change this id to be more distinct. We had issues in the past because of the svgo rule that minimizes ids (it's disabled in DS but probably is not in upstream).
```
id="logo-fandom-powered-by-wikia-gradient"
```